railties/lib/rails/generators/rails/app/templates/config/boot.rb.tt

```diff
@@ -1,3 +1,7 @@
+<% unless options.skip_spring? -%>
+load File.expand_path("../bin/spring", __dir__)
```

Comment:
I left out the [`RAILS_ENV` check](https://github.com/rails/rails/pull/39225/files#diff-597f6e4e2781654a9c071984ba0940adR1) based on https://github.com/rails/rails/pull/39225#issuecomment-632855297.